src/commands/cloud.rs

```diff
@@ -0,0 +1,24 @@
+use anyhow::Result;
+use clap::Subcommand;
+
+use super::deploy::DeployCommand;
+use super::login::LoginCommand;
+
+/// Commands for publishing applications to Fermyon Cloud.
+#[derive(Subcommand, Debug)]
+pub enum CloudCommands {
+    /// Package and upload an application to Fermyon Cloud.
```

Comment:
Nit: despite the new naming, this works with the Platform as well - should we say e.g. "to Fermyon" (could be confusing for Fermyon the product vs Fermyon the company) or "to Fermyon Cloud or the Fermyon Platform"?